coalib/processes/Processing.py

```diff
@@ -405,13 +405,21 @@ def instantiate_processes(section,
                         'control_queue': control_queue,
                         'timeout': 0.1}
 
+    loaded_local_bears_count = len(local_bear_list)
     local_bear_list[:], global_bear_list[:] = instantiate_bears(
         section,
         local_bear_list,
         global_bear_list,
         complete_file_dict,
         message_queue,
         console_printer=console_printer)
+    loaded_valid_local_bears_count = len(local_bear_list)
+
+    if(loaded_local_bears_count != loaded_valid_local_bears_count):
+        if cache:
+            log_printer.debug('The cached files have been reset because one of'
+                              'the bears didn\'t meet it\'s requirements.')
```

Comment:
@impmihai actually the preferred quotation style is to use single quotes... but you can use double quotes for lines having single quotes in them... I had a similar discussion sometime back with @fneu and @nemaniarjun ...  
check this https://github.com/coala/coala-bears/pull/1236#discussion_r94857243